### PR TITLE
fix(form): el-from/src/token.d.ts file miss (#2979)

### LINKED ISSUE #2979: [Bug Report] el-from/src/token.d.ts file miss
### Element Plus version
1.0.2-beta.71

### OS/Browsers version
windows10/vscode/yarn

### Vue version
3.1.5

### Reproduction Link
https://elementui.github.io/issue-generator/#/en-US?repo=element-plus

### Steps to reproduce
yarn add element-plus
open file: node_modules\element-plus\lib\el-form\index.d.ts
@line 5: export * from './src/token';

### What is Expected?
open file: node_modules\element-plus\lib\el-form\index.d.ts
@line 5: export * from './src/token';
the file: el-from/src/token.d.ts is in element-plus npm package 

### What is actually happening?
open file: node_modules\element-plus\lib\el-form\index.d.ts
@line 5: export * from './src/token';
the file: el-from/src/token.d.ts in element-plus npm package is miss

### PR DESCRIPTION
**Pull request by imswk**
fix #2979

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/form/src/token.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export interface ValidateFieldCallback {
 export interface ElFormItemContext {
   prop?: string
   formItemMitt: Emitter
-  size: ComponentSize
+  size?: string
```
Comment (JeremyWuuuuu) [inline packages/form/src/token.ts:33]: We should import `ComponentSize` from `@element-plus/utils/types` instead

Comment (imswk) [reply packages/form/src/token.ts:33]: Okay~

Comment (imswk) [reply packages/form/src/token.ts]: @JeremyWuuuuu

File: packages/form/src/token.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import type { Emitter } from 'mitt'
 import type {
   FieldErrorList,
 } from 'async-validator'
+import { ComponentSize } from '@element-plus/utils/types'
```
Comment (JeremyWuuuuu) [inline packages/form/src/token.ts:6]: `import type { ComponentSize }`

File: packages/form/src/token.ts (hunk 1 above)
Comment (JeremyWuuuuu) [reply packages/form/src/token.ts]: @imswk Hi, I left a comment, please address that so we can get this merged.

File: packages/form/src/token.ts (hunk 2 above)
Comment (imswk) [reply packages/form/src/token.ts]: @JeremyWuuuuu
I'm done. 🙇

**Review by jw-foss — APPROVED** [review]

**Pull request merged**